### PR TITLE
es6-destructuring - Cindy Le

### PR DESCRIPTION
**Pull request by cindy-le91**
Screen Shot 2023-05-08 at 10 24 25 AM

### CODE REVIEW

File: es6-destructuring/index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        libraryID: 3245
+      },
+      {
+        title: "Dead Dead Demon's De De De De Destruction",
+        author: 'Inio Asano',
+        libraryID: 1233
+      }
+    ];
+
+    const book3 = library[0];
```
Comment (robgmerrill) [inline es6-destructuring/index.html:50]: Lines 50, 51, 52 are not using the destructuring technique. Please review after Q&A :)

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Great start! Destructuring is not being applied in a few places and will need to be revisited.

**Comment by cindy-le91** [comment]
Screen Shot 2023-05-08 at 11 37 59 AM

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**